src/styles/components/blogHeader.scss

```diff
@@ -30,11 +30,6 @@
       margin-right: 2rem;
       margin-bottom: 0.25rem;
 
```

Comment:
make sure the alignment of last `span` element of blogHeader in /blog/ is correct